### ISSUE TITLE
How to stop page from loading

### ISSUE THREAD

**Issue by Paintras** [Codeception/Codeception#4454]
Good day,
I have this code:
```php
$aLinkText = $I->grabMultiple('a','href');
for($x = 12; $x < count($aLinkText); $x++)
{
	try{
	print($x . "\n");
	$URL = $aLinkText[$x];
	$I->amOnURL($aLinkText[$x]);
	$I->dontSee("Notice");
	$I->wait(1);
	}
	catch(Exception $ex)
	{
		print("There was an exception thrown at: \t");
		print($URL);
		print("\n");
		continue;
	}
}
```
I will grab all <a> elements and their href attribute and call $I->amOnURL() to run throught all of them,  When it get to the exception ( timeout or etc ) i want it to continue in cycle. It actually continues, but the previous page is still loading ( there is nette bluescreen exception loading ) and if it does not load until timeout it would say the next URL was also having exception.
The question is. Is there some webdriver command that I could use to stop page from loading (when it get to the exception) so it can call next $I->amOnURL() ? Simplified the previous for iteration is affecting the next one.
Thanks for possible help

**Comment by Naktibalda** [comment]
What's the error which is returned while the page is still loading?

1. You should probably use `cantSee` instead of `dontSee` and try catch block.
2. You could use `_restart` method to restart browser at the beginning of each iteration. (You have to create a helper method to be able to use it).

**Comment by Paintras** [comment]
Thank you for your suggestion. Yeah i have used custom method inside catch block and inside it called _closeSession() and _restart() and works just fine for me. Would try cantSee instead of dostSee.
For the error: the problem was I got small timeout for page to load. When I catched an exception the page was still loading and then next one got timeoutException as well. _restart() solved it and upon calling next amOnURL() i have seprated it from the previous loading URL. 
Closing issue.
